js/dimensions.js

```diff
@@ -136,8 +147,15 @@ $(document).ready(function() {
 
   $(document).on('change', '#dimension_image_upload', function() { Resize.readFile(this); });
   $(document).on('change', '#resize-select', function() { Resize.fillDimensionFields(this); });
-  $(document).on('click', '.preview-result', function() { Resize.showResult(); });
   $(document).on('click', '.submit-btn', function(event) { Resize.downloadableResult(); });
+  $(document).on('click', '.preview-result', '#user-width, #user-height', function() {
```

Comment:
Why are the 4 arguments here? What is the third argument?

Reply:
I though I would need the third argument to pass in the width and height dimensions selected by the user when cropping. However I tried it just now without the third argument and it worked just fine, so I'll remove it in the next commit.

As a side note, I think that the line above with the `submit-btn` and `Resize.downloadableResult()` is old code cause I never found either of those mentioned anywhere else in the html or javascript. My guess is that it was supposed to be replaced by the one I modified, but I'm not 100% sure.